# header

Header header

# Number of keypoints in frame

uint32 num

# 2D image-frame coordinates

float32[] x_img
float32[] y_img

# 3D world coordinates

float32[] x_w
float32[] y_w
float32[] z_w